tests/integration/test_cubepart.py

```diff
@@ -1181,6 +1208,21 @@ def it_provides_sum_and_mean_measure_for_CAT(self):
         assert strand.means == pytest.approx([2.66666667, 3.5])
         assert strand.sums == pytest.approx([8, 7])
 
+    def it_provides_share_of_sum_measure_for_CAT(self):
+        strand = Cube(CR.CAT_SUM).partitions[0]
+
+        assert strand.sums == pytest.approx([88.0, 77.0])
+        # --- share of sum is the array of sum divided by its sum, so in this case
+        # --- [88/165, 77/165]
+        assert strand.share_sum == pytest.approx([0.5333333, 0.4666666])
```

Comment:
I think you could use `strand.share_sum.tolist()` and then just compare to the plain list (but maybe you'd have to make longer literals for numbers). `pytest.approx` is also pretty long itself, so, see which one fits better...